product_docs/docs/pgd/5/cli/discover_connections.mdx

```diff
@@ -51,33 +55,41 @@ instances:
+The connection string for this cluster is:
 
 ```
 "host=192.168.100.2 port=6432 dbname=bdrdb user=enterprisedb sslmode=require"
 ```
 
 !!! Note Host name versus IP address
-In our example, we use the IP address because the configuration is from a Docker TPA install with no name resolution available. Generally, you should be able to use the host name as configured.
+The example uses the IP address because the configuration is from a Docker TPA install with no name resolution available. Generally, you can use the host name as configured.
 !!!
 
 ### For a BigAnimal distributed high-availability cluster
 
-1. Log into the [BigAnimal Clusters](https://portal.biganimal.com/clusters) view.
+1. Log in to the [BigAnimal clusters](https://portal.biganimal.com/clusters) view.
 1. Select your cluster.
-1. In the view of your cluster, select the Connect tab.
-1. Copy the Read/Write URI from the connection info. This is your connection string.
+1. In the view of your cluster, select the **Connect** tab.
+1. Copy the read/write URI from the connection info. This is your connection string.
+
+### For a cluster deployed with EDB PGD for Kubernetes
+
+As with TPA, EDB PGD for Kubernetes is very flexible, and there are multiple ways to obtain a connection string. It depends, in large part, on how the [services](/postgres_distributed_for_kubernetes/latest/connectivity/#services) were configured for the deployment: 
+
+- If you use the Node Service Template, direct connectivity to each node and proxy service is available. 
+- If you use the Group Service Template, there's a gateway service to each group. 
+- If you use the Proxy Service Template, a single proxy provides an entry point to the cluster for all applications. 
 
-### For an EDB PGD for Kubernetes deployed cluster
+Consult your configuration file to determine this information. 
 
-As with TPA, EDB PGD for Kubernetes is very flexible and there is no one way to obtain a connection string. It depends, in large part, on how the [Services](https://www.enterprisedb.com/docs/postgres_distributed_for_kubernetes/latest/connectivity/#services) have been configured for the deployment. If the Node Service Template is used, there should be direct connectivity to each node and proxy service available. If the Group Service Template, there will be a gateway service to each group. Finally, if the Proxy Service Template has been used, there should be a single proxy providing an entry point to the cluster for all applications. Consult your configuration file to determine this information. You should be able to establish a host name or IP address, port, database name (default: `bdrdb`) and username (`enterprisedb` for EPAS and `postgres` for Postgres and Postgres Extended.).
+Establish a host name or IP address, port, database name, and username. The default database name is `bdrdb`, and the default username is enterprisedb for EDB Postgres Advanced Server and postgres for Postgres and Postgres Extended.).
```

Comment:
again i think that enerprisedb and postgres should be enclosed in pics similar to bdrdb

Reply:
```suggestion
Establish a host name or IP address, port, database name, and username. The default database name is `bdrdb`, and the default username is `enterprisedb` for EDB Postgres Advanced Server and `postgres` for Postgres and Postgres Extended.).
```

For clarity, they are called backticks or backquotes btw, not pics.